src/main/scala/com/databricks/spark/redshift/RedshiftJDBCWrapper.scala

```diff
@@ -151,9 +151,9 @@ private[redshift] class JDBCWrapper {
    * @throws SQLException if the table contains an unsupported type.
    */
   def resolveTable(conn: Connection, table: String): StructType = {
-    val rs = executeQueryInterruptibly(conn.prepareStatement(s"SELECT * FROM $table WHERE 1=0"))
+    val ps = conn.prepareStatement(s"SELECT * FROM $table WHERE 1=0")
```

Comment:
Note: it's still important to leave the `WHERE 1=0` clause in order to limit the work of the query in case the underlying JDBC driver implementation happens to implement this by actually executing the query (some abstract JDBC driver implementations do this; it looks like Redshift and Postgres don't but I'd like to leave this code here as a safety-net in case older drivers do something silly).